ansible/roles/azure/tasks/terminate_cluster.yml

```diff
@@ -19,6 +19,38 @@
 
 - hosts: localhost
   tasks:
+    - block:
+      - import_tasks: log_analytics_ws_common.yml
+
+      - name: Delete workbook
+        azure_rm_resource:
+           resource_group: "{{ resource_group }}"
+           provider: insights
+           resource_type: workbooks
+           resource_name: "{{ workbook_exists|join('') }}"
+           api_version: '2020-02-12'
+           state: absent
+        when: workbook_exists is defined
+
+      - name: Delete dashboard
+        azure_rm_resource:
+           resource_group: "{{ resource_group }}"
+           provider: Portal
+           resource_type: dashboards
+           resource_name:  "{{ monitor_name.stdout + '-db' }}"
+           api_version: '2019-01-01-preview'
+           state: absent
+
+      - name: Delete log analytics workspace
+        azure_rm_resource:
+           resource_group: "{{ resource_group }}"
+           provider: OperationalInsights
+           resource_type: workspaces
+           resource_name: "{{ monitor_name.stdout + '-la' }}"
```

Comment:
Could `{{ log_workspace_name }}` be used here?

Reply:
Thanks for catching that. Made the required change